ui_framework/components/local_nav/_local_search.scss

```diff
@@ -44,6 +48,15 @@
     transform: translateY(-50%); /* 1 */
   }
 
+  .kuiLocalSearchAssistedInput__assistanceLink {
+    display: none;
+  }
+
+  .kuiLocalSearchInput:focus + .kuiLocalSearchAssistedInput__assistance .kuiLocalSearchAssistedInput__assistanceLink,
+  .kuiLocalSearchAssistedInput__assistanceLink:active {
+    display: inherit;
+  }
+
```

Comment:
Can we replace lines 31 through 69 with this:

```scss
/**
 * 1. Visually hide this until the parent is focused, so that it's accessible to screen readers.
 */
.kuiLocalSearchInput:focus {
  & ~ .kuiLocalSearchAssistedInput__assistance {
    .kuiLocalSearchAssistedInput__assistanceLink {
      opacity: 1; /* 1 */
    }
  }
}

.kuiLocalSearchAssistedInput {
  display: flex;
  flex: 1 1 100%;
  position: relative;
}

  /**
   * 1. Vertically center the assistance, regardless of its height.
   */
  .kuiLocalSearchAssistedInput__assistance {
    position: absolute;
    right: $globalFormControlHorizontalPadding;
    top: 50%; /* 1 */
    transform: translateY(-50%); /* 1 */
  }

  /**
   * 1. Visually hide this until it's focused, so that it's accessible to screen readers, and is
   *    also keyboard-accessible.
   */
  .kuiLocalSearchAssistedInput__assistanceLink {
    opacity: 0; /* 1 */

    &:active,
    &:focus {
      opacity: 1; /* 1 */
    }
  }
```

These changes will ensure the link is visible to screen readers, and can also be tabbed to by users who are using the keyboard to navigate.